ecosystem/python/sdk/pyproject.toml

```diff
@@ -11,9 +11,9 @@ keywords = ["web3", "sdk", "aptos", "blockchain"]
 
 [tool.poetry.dependencies]
 h2 = "^4.1.0"
-httpx = "^0.23.0"
```

Comment:
Hmm, looks like this is used by async_client. Will add it back. But interesting that the tests still pass?

Reply:
Added it back. Confirmed that things would break, but only after running `poetry install --sync`